wiki/forms.py

```diff
@@ -1,22 +1,24 @@
 # -*- coding: utf-8 -*-
+import re
+from itertools import chain
+
 from django import forms
-from django.utils.translation import ugettext_lazy as _
-from django.utils.safestring import mark_safe
+from django.contrib.auth.models import User
 from django.forms.utils import flatatt
+from django.forms.widgets import HiddenInput
 from django.utils.encoding import force_unicode
-from django.utils.html import escape, conditional_escape, strip_tags
-
-from itertools import chain
+from django.utils.html import conditional_escape, escape, strip_tags
+from django.utils.safestring import mark_safe
+from django.utils.translation import ugettext_lazy as _
 
 from wiki import models
 from wiki.conf import settings
-from wiki.editors import getEditor
-from wiki.core.diff import simple_merge
-from django.forms.widgets import HiddenInput
-from wiki.core.plugins.base import PluginSettingsFormMixin
-from django.contrib.auth.models import User
 from wiki.core import permissions
 from wiki.core.compat import BuildAttrsCompat
+from wiki.core.diff import simple_merge
+from wiki.core.plugins.base import PluginSettingsFormMixin
+from wiki.editors import getEditor
```

Comment:
You reordered imports... is it isort of your pycharm?

Reply:
Yes, you're right. I reordered it because I added import re. But according your comments it could be realized without regular expression, so I will revert changes according to imports